yt/data_objects/tests/test_center_squeeze.py

```diff
@@ -6,7 +6,7 @@ def test_center_squeeze():
 
     # create and test amr, random and particle data
     check_single_ds(fake_amr_ds(fields=("Density",)))
-    check_single_ds(fake_random_ds(16, fields=("Density",)))
+    check_single_ds(fake_random_ds(16, fields=("Density",), units=("g/cm**3",)))
```

Comment:
@matthewturk I think I made a mistake here... "Density" is supposed to be a number density in `"cm**(-3)"`, right ?